test/src/loaders/workspace_uploader/workspace_uploader_test.py

```diff
@@ -227,18 +236,67 @@ def test_post_process(setup_and_teardown):
                                "genome_upa": None,
                                "genome_filename": None}}}}
 
-    dest_file = os.path.join(
-        os.path.join(output_dir, "12345_58_1"), f"12345_58_1.fna.gz"
+    assert uploaded
+    assert expected_data == data
+
+    # test with genome_tuple and genome_upa
```

Comment:
This seems like it should be a separate test

Reply:
👍

Reply:
:+1: